executor/executor.py

```diff
@@ -123,6 +123,33 @@ def update(
 
         self._index.update(flat_docs)
 
+    @requests(on='/filter')
+    def filter(
```

Comment:
With the goal of filtering, I think we can directly use the `filter()` function implemented in annlite.
```
@requests(on='/filter')
def filter(self, ....):
    conditions = ....
    return self._index.filter(filter=conditions)
```